images/wkdev_sdk/Containerfile

```diff
@@ -127,8 +137,12 @@ RUN export QT_VERSION=$(qmake6 -query QT_VERSION) && \
       ln -s ${directory} ${directory}/${QT_VERSION}  >/dev/null 2>&1 || true; \
     done
 
-# Build custom sccache version (the Ubuntu provided 0.7.7 package is broken)
-RUN cargo install --root /usr/local --version 0.8.1 --locked sccache
+# Check GStreamer plugins are installed.
+RUN gst-inspect-1.0 audiornnoise && \
+    gst-inspect-1.0 cea608tott && \
+    # gst-inspect-1.0 dav1ddec && \ FIXME(dpino): Not installed.
```

Comment:
`daviddec` 0.13 supports only the most recent libdav1d (iirc 1.2.x). So you might need to add libdav1d in jhbuild too.